.github/workflows/test.yml

```diff
@@ -41,24 +41,20 @@ jobs:
           mamba install --file=requirements.txt --file=requirements-testing.txt
           mamba list
 
-      - name: Install Windows requirements
-        if: matrix.os == 'windows-latest'
-        run: |
-          mamba install pymt_child pymt_hydrotrend pymt_permamodel
+      # - name: Install Windows requirements
+      #   if: matrix.os == 'windows-latest'
+      #   run: |
+      #     mamba install pymt_child pymt_hydrotrend pymt_permamodel
 
-      - name: Install Unix requirements
-        if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
-        run: |
-          mamba install esmpy
-          mamba install pymt_cem pymt_child pymt_hydrotrend pymt_permamodel
+      # - name: Install Unix requirements
+      #   if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      #   run: |
+      #     mamba install pymt_cem pymt_child pymt_hydrotrend pymt_permamodel
```

Comment:
What's the reason for not installing these components? I assume they're not working for some reason?

Reply:
#164 is the problem. I revert this in e68392b094e84ab9b8a74638946ef05e8e774976, in a branch, https://github.com/mdpiper/pymt/tree/mdpiper/fixes-for-shapely-and-scipy-updates, that I was planning to submit as a PR after this one.